common/css-paint-tests.js

```diff
@@ -0,0 +1,23 @@
+// We call rAF and in the second frame we compare the result with the ref.html.
+function importPaintWorkletAndTerminateTestAfterAsyncPaint(code) {
+    if (window.testRunner) {
+        testRunner.waitUntilDone();
```

Comment:
These are WebKit/Blink internals and shouldn't be included in wpt. The `class="reftest-wait"` mechanism is probably what's needed, so common/reftest-wait.js for a script wrapping that.